### DISCUSSION TITLE
元数据存储使用mysql时没有自动创建表

### DISCUSSION THREAD

**Discussion by TristeLXH** [milvus-io/milvus#26545] [Q&A and General discussion] [answered]
我使用的milvus:v2.2.13版本，在milvus.yaml文件中，我发现一个关于元数据存储配置项：
# Default value: etcd
# Valid values: [etcd, mysql]
metastore:
  type: etcd

# Related configuration of mysql, used to store Milvus metadata.
mysql:
  username: root
  password: 123456
  address: localhost
  port: 3306
  dbName: milvus_meta
  driverName: mysql
  maxOpenConns: 20
  maxIdleConns: 5
  
  但是我配置为mysql之后，启动milvus报错提示没有对应的表
  而且我在文档中也没有找到对应的建表语句，请问从哪儿能看到建表语句

**Comment by xiaofan-luan** [comment, marked as answer]
currently we no longer support mysql as meta store. sorry for the misunderstanding

**Reply by TristeLXH** [reply]
明白了，感谢

**Comment by xiaofan-luan** [comment]
@jaime0815 
please help on removing the legacy code
